File: python-sdk-examples/completion/text.py
from writerai import Writer
import dotenv

dotenv.load_dotenv()

client = Writer()

def main():
    try:
        completion = client.completions.create(
            model="palmyra-x-003-instruct",
            prompt="Tell me a joke about Python",
            temperature=0.7,
            stream=False
        )

        print(completion.choices[0].text)
        
    except Exception as error:
        print("Error:", error)

if __name__ == "__main__":
    main()

File: node-sdk-examples/completion/text.js
const Writer = require('writer-sdk');
const dotenv = require('dotenv');

dotenv.config();

const client = new Writer({
    apiKey: process.env.WRITER_API_KEY,
});

async function main() {
    try {
      const completion = await client.completions.create({
        model: 'palmyra-x-003-instruct',
        prompt: 'Tell me a story about a magical forest and a hero who must save the forest from a dark wizard.',
        temperature: 0.7
      });

      console.log(completion.choices[0].text);
    } catch (error) {
      console.error('Error:', error);
    }
  }

main();